tests/unit/helpers/surcharges.test.js

```diff
@@ -84,127 +84,320 @@ describe('delete', () => {
 });
 
 describe('getCustomSurcharge', () => {
-  it('should return null if there is no surcharge', () => {
-    const res = SurchargesHelper.getCustomSurcharge('2018-01-01T05:00:00', '2018-01-01T09:00:00', '05:00', '13:00');
+  it('should return [] if there is no surcharge', () => {
+    const eventStart = '2018-01-01T05:00:00';
+    const eventEnd = '2018-01-01T09:00:00';
+    const res = SurchargesHelper.getCustomSurcharge(eventStart, eventEnd, '05:00', '13:00');
```

Comment:
Que penses-tu de plutot le nommer "should return [] if there is no percentage surcharge", pcq au debut je pensais qu'on testait le cas ou il n'y a pas de majoration sur la periode de l'evenement

Reply:
done